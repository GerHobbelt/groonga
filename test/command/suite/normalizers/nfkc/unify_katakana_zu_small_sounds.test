#@add-substitution /"version",\s"(?:.+?)"/ "\"version\", \"#{ENV['NFKC'] || '10.0.0'}\"" "\"version\", \"NFKC_VERSION\""
normalize \
  'NormalizerNFKC("unify_katakana_zu_small_sounds", true, \
                  "report_source_offset", true, \
                  "version", "NFKC_VERSION")' \
  "ズァズィズズェズォ" \
  WITH_CHECKS|WITH_TYPES
#@remove-substitution /"version",\s"(?:.+?)"/
