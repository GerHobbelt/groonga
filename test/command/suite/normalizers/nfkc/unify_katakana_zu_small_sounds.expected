normalize   'NormalizerNFKC("unify_katakana_zu_small_sounds", true,                   "report_source_offset", true,                   "version", "NFKC_VERSION")'   "ズァズィズズェズォ"   WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ザジズゼゾ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null"
    ],
    "checks": [
      6,
      0,
      0,
      6,
      0,
      0,
      3,
      0,
      0,
      6,
      0,
      0,
      6,
      0,
      0
    ],
    "offsets": [
      0,
      6,
      12,
      15,
      21,
      27
    ]
  }
]
